page/index.md

```diff
@@ -22,30 +22,30 @@ Check out our next get together on **December 13th 2023**!
 
 \begin{section}{title="Upcoming Events"}
 
-### Introduction to Julia
+### Julia at Murata
 
-Would you like to know more about Julia? Come to our particularly beginner friendly introduction!
+Learn how Murata uses Julia in their work! **Register by TBA**
```

Comment:
maybe a nicer description?

Reply:
@jaakkor2, Maybe add a one-liner explaining what is Murata core business; I suspect some won't know the company. I would also consider the possibility of attending remotely (including for myself 😄)

Reply:
fwiw, we discussed with Jaakko to move this to April 9.4 to have more time to advertise